README.md

```diff
@@ -108,6 +108,16 @@ Feel free to adjust the command line arguments to your needs:
   * `export` create a data export for the specified countries in `csv` or `geo-json` format
 * `online` fetch data online from original data sources, if `false` use files cached on disk
 
+
+Before re-running merge w/o re-import, delete corresponding DB entries
+
+    delete from echarm_merged_station_source ;
```

Comment:
A hint that "echarm_" is a DB_TABLE_PREFIX is missing.   
I think we can also skip this part because once you create an PR with the planned command "clean_tables" the listing here is obsolete imo.

Reply:
Yes, will clean it up, if it's not already removed in the other PR